### DISCUSSION TITLE
False positive hand output

### DISCUSSION THREAD

**Discussion by YaraAmin** [vladmandic/human#202] [Q&A]
Hi
I'm using human on stream, I'm getting like 4-6 frames per second, and this is my configs for Human:
```
HumanConfig = {
  backend: 'humangl',
  debug: false,
  async: true,
  cacheSensitivity :  0, 
  filter: { enabled: false },
  face: {
    enabled: true,
    detector: { enabled: true, modelPath: 'blazeface.json' , rotation: false, maxDetected: 1 },
    mesh: { enabled: true, modelPath: 'facemesh.json' },
    iris: { enabled: true, modelPath: 'iris.json' },
    description: { enabled: true, modelPath: 'faceres.json', minConfidence: 0.1 },
    emotion: { enabled: true, modelPath: 'emotion.json', minConfidence: 0.1},
  },
  hand: { enabled: true, maxDetected: 1, minConfidence: 0.5, detector: { modelPath: 'handtrack.json' } },
  body: { enabled: true , modelPath: 'movenet-lightning.json',minConfidence: 0.2},
  object: { enabled: false},
};
```
but I'm getting a lot of false-positive hand data where it detects that there's a hand with less than 0.75 confidence score when there's no hand in the frame. I thought of a higher confidence score for the hand model but when there is a hand it detects it with the same confidence score! 
I'm using the 2.4.2 version 

Do you recommend anything to overcome this issue? thanks!

**Comment by vladmandic** [comment]
This is better suited as an issue, not a discussion item - created issue #203 to track.

**Comment by ubilrodriguez** [comment]
> Hola , estoy usando humanos en la transmisión, obtengo como 4-6 cuadros por segundo, y esta es mi configuración para Human:
> 
> ```
> HumanConfig = {
>   backend: 'humangl',
>   debug: false,
>   async: true,
>   cacheSensitivity :  0, 
>   filter: { enabled: false },
>   face: {
>     enabled: true,
>     detector: { enabled: true, modelPath: 'blazeface.json' , rotation: false, maxDetected: 1 },
>     mesh: { enabled: true, modelPath: 'facemesh.json' },
>     iris: { enabled: true, modelPath: 'iris.json' },
>     description: { enabled: true, modelPath: 'faceres.json', minConfidence: 0.1 },
>     emotion: { enabled: true, modelPath: 'emotion.json', minConfidence: 0.1},
>   },
>   hand: { enabled: true, maxDetected: 1, minConfidence: 0.5, detector: { modelPath: 'handtrack.json' } },
>   body: { enabled: true , modelPath: 'movenet-lightning.json',minConfidence: 0.2},
>   object: { enabled: false},
> };
> ```
> 
> pero obtengo una gran cantidad de datos de manos falsos positivos en los que detecta que hay una mano con un puntaje de confianza inferior a 0,75 cuando no hay una mano en el marco. Pensé en una puntuación de confianza más alta para el modelo de mano, pero cuando hay una mano, ¡la detecta con la misma puntuación de confianza! Estoy usando la versión 2.4.2
> 
> ¿Recomiendas algo para solucionar este problema? ¡Gracias!

sorry for the ignorance, but where is this file located? can you give me the path of this file in the project

**Reply by vladmandic** [reply]
@ubilrodriguez that is a user defined configuration, not a file to be found. read documentation.

**Reply by ubilrodriguez** [reply]
sorry for the ignorance, but where is the documentation?

**Comment by vladmandic** [comment]
sorry, but WTF?! its on a home page! <https://github.com/vladmandic/human> 
and all subsection are listed there.
